hnn_core/viz.py

```diff
@@ -186,8 +186,12 @@ def plot_spikes_raster(cell_response, ax=None, show=True):
             cell_type_ypos.append(np.repeat(ypos, len(gid_time)))
             ypos = ypos - 1
 
-        cell_type_times = np.concatenate(cell_type_times)
-        cell_type_ypos = np.concatenate(cell_type_ypos)
+        if cell_type_times:
+            cell_type_times = np.concatenate(cell_type_times)
+            cell_type_ypos = np.concatenate(cell_type_ypos)
+        else:
+            cell_type_times = []
+            cell_type_ypos = []
```

Comment:
maybe add a tiny smoke test? is it possible?

Reply:
Might be awkward to write since a you'd have to access figure elements to check if spikes were plotted correctly. Do you have anything specific in my mind?

Reply:
yeah I know. That's why I suggested a "smoke test". It's where you don't check the output but simply check that it runs without throwing an error. On master, it throws an error, right?

Reply:
Ah I see. Perhaps just grab the output from a test simulation that throws no spikes? I'll check if that's the case for any existing tests. If not it shouldn't contribute significantly to testing time to run a really short simulation.

Reply:
yes exactly!